### DISCUSSION TITLE
How to set UART buffer size in boards.txt

### DISCUSSION THREAD

**Discussion by specternecter** [SpenceKonde/megaTinyCore#583] [Q&A]
In version 2.5.1 I could set the UART buffer size to 256 via defining them in the variants and setting the USE_ASMs to zeros in UART.h. This doesn't work in 2.5.4 and it says that buffer size can be overwritten in boards.txt, but I don't know how to do that. Is there an easy way or a future version where this can be selected by the IDE in the way other options are?

**Comment by SpenceKonde** [comment]
o_o That should still work... What error do you get? 

You can also set the USE_ASM's to 0 in the variant file....

**Reply by specternecter** [reply]
It turns out it was just an update issue. serialupdi didn't work either which didn't make sense, so I manually deleted megaTinyCore and then installed 2.5.4. It works fine now
